mars/utils.py

```diff
@@ -1766,3 +1767,20 @@ def retry_call(*args, **kwargs):
             raise ex  # pylint: disable-msg=E0702
 
     return retry_call
+
+
+def add_aiotask_done_check_callback(
+    aiotask: asyncio.Task, exception_formatter: str, *args
+):
+    def _on_mars_aiotask_done(fut):
+        # Print the error of task.
+        try:
+            fut.result()
+        except asyncio.CancelledError:  # pragma: no cover
+            pass
+        except Exception:  # pragma: no cover
+            logger.exception(exception_formatter, *args)
+            if _is_ci:
+                sys.exit(-1)
+
+    aiotask.add_done_callback(_on_mars_aiotask_done)
```

Comment:
Can we use `mars.oscar.debug.create_task_with_error_log` for print exception?

Reply:
I didn't notice the `mars.oscar.debug.create_task_with_error_log`. Yes, I can use it. Thanks.